pkg/promclient/promclient.go

```diff
@@ -376,6 +379,32 @@ func PromqlQueryInstant(ctx context.Context, logger log.Logger, base *url.URL, q
 	return vec, warnings, nil
 }
 
+// GetPromVersion will return the version of Prometheus by querying /version Prometheus endpoint.
```

Comment:
Function name doesn't follow documentation, consider renaming function to `GetPromVersion` or changing doc comment to `PromVersion`

Reply:
Done.